### PR TITLE
Add Microsoft.Data.Sqlite snippets

### PR DESCRIPTION
**Pull request by bricelam**
cc @mairaw

### CODE REVIEW

**Comment by bricelam** [comment]
cc @ajcvickers

File: snippets/standard/data/sqlite/AggregateFunctionSample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                "stdev",
+
+                // A tuple to maintain context between rows
+                (Count: 0, Mean: 0.0, Sum: 0.0),
+
+                // This is called for each row
+                ((int Count, double Mean, double Sum) context, double value) =>
+                {
+                    context.Count++;
+                    var delta = value - context.Mean;
```
Comment (BillWagner) [inline snippets/standard/data/sqlite/AggregateFunctionSample/Program.cs]: I think this is mathematically wrong. The `context.Mean` gets updated each row, so the `delta` would be accurate only for the final row.  

See [this example](https://docs.microsoft.com/en-us/dotnet/csharp/tuples#tuples-as-method-return-values) for an alternative algorithm that uses one enumeration for StdDev.

Comment (bricelam) [reply snippets/standard/data/sqlite/AggregateFunctionSample/Program.cs]: Copied the example's algorithm. The old result is definitely incorrect. Thanks! lol, that's what I get for grabbing some arbitrary algorithm off Wikipedia... 😉

**Comment by BillWagner** [comment]
Thanks for making those updates @bricelam This is ready to :shipit:

**Comment by mairaw** [comment]
@bricelam do you have permissions to squash and merge? Otherwise, we'll do it for you.

**Comment by bricelam** [comment]
No permissions

**Comment by BillWagner** [comment]
It's merged @bricelam

**Pull request merged**